bin/test-instruction-logger/SetGithubCredentialsCommand.php

```diff
@@ -0,0 +1,63 @@
+<?php // phpcs:ignore WordPress.Files.FileName.NotHyphenatedLowercase
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
```

Comment:
@moon0326 The `symfony/console` composer package should probably be added as a dev dependency because it's not currently declared. It looks like it's just included via the `jetpack-changelogger` package at the moment.

```
composer require --dev symfony/console
```

What do you think?